### ISSUE TITLE
Advanced search dropdowns have no accessible label

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/SearchWorks#3517]
The advanced search has two dropdowns — one at the top and one at the bottom — that don't have an accessible label.

This one (inside a heading):

<img width="314" alt="Screenshot 2023-09-18 at 13 03 22" src="https://github.com/sul-dlss/SearchWorks/assets/4924494/56780b56-3c6b-45f3-b965-56f119495b32">

And this one (on its own):

<img width="451" alt="Screenshot 2023-09-18 at 13 04 49" src="https://github.com/sul-dlss/SearchWorks/assets/4924494/7d9f8200-58cc-4fcd-b5ea-e96269b96f9c">

Normally you'd want to have regular `<label>`s wrapping the select or associated with it using `for=`, but if we want to be minimally invasive to this page structure, we could consider just adding `aria-label`s, which will be otherwise invisible.

Naming the first dropdown is hard — putting it inside other text is sort of clever, but makes it hard for assistive tech to understand. We might consider taking it out of the heading and giving it a regular label.

This is a [WCAG Level A violation](https://www.w3.org/TR/WCAG20-TECHS/F77.html), which is something SODA has asked us to remediate whenever time allows.

[See this issue in SiteImprove](https://my2.siteimprove.com/Accessibility/1059585/NextGen/Issue/1?ruleId=8&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=21&siteTarget.wcagVersion=21&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2)

**Comment by thatbudakguy** [comment]
Could we realistically get to modern BL this fall? I agree that that's the ideal solution, but it seems like we could resolve the issue with a much quicker stopgap for now

**Comment by jcoyne** [comment]
@thatbudakguy yes, why not? We'd just need to put it on the board.

**Comment by thatbudakguy** [comment]
blocked by bs4 upgrade and maybe also blacklight upgrade

**Comment by thatbudakguy** [comment]
Resolved by bootstrap 4 upgrade
